cwms-http-client/src/main/java/mil/army/usace/hec/cwms/http/client/SslSocketData.java

```diff
@@ -0,0 +1,42 @@
+package mil.army.usace.hec.cwms.http.client;
+
+import java.util.Objects;
+import javax.net.ssl.SSLSocketFactory;
+import javax.net.ssl.X509TrustManager;
+
+public final class SslSocketData {
+
+    private final SSLSocketFactory sslSocketFactory;
+    private final X509TrustManager x509TrustManager;
+
+    public SslSocketData(SSLSocketFactory sslSocketFactory, X509TrustManager x509TrustManager) {
+        this.sslSocketFactory = Objects.requireNonNull(sslSocketFactory, "Missing required SSLSocketFactory");
+        this.x509TrustManager = Objects.requireNonNull(x509TrustManager, "Missing required X509TrustManager");
+    }
+
+    SSLSocketFactory getSslSocketFactory() {
+        return sslSocketFactory;
+    }
+
+    X509TrustManager getX509TrustManager() {
+        return x509TrustManager;
+    }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Why are equals/hashcode implemented and dependent on SslSocketFactory and CwbiAuthTrustManager when they don't implement the methods themselves?

Reply:
This doesn't need to implement equals/hashcode anymore. Removed those methods.